boom-app/src/components/DisplayComponent.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import type { CsvOutput } from '@/types'
+
+defineProps<{ csvData: CsvOutput | undefined }>()
+</script>
+
+<template>
+  <h1>Observe that uploaded data!</h1>
+  <div class="data-box">
+    <table>
+      <tr>
+        <th v-for="(headerName, index) in csvData?.headers" :key="index">
+          {{ headerName }}
+        </th>
+      </tr>
+      <tr v-for="(item, index) in csvData?.data" :key="index">
+        <td v-for="(headerName, index) in csvData?.headers" :key="index">
+          {{ item[headerName] }}
+        </td>
+      </tr>
+    </table>
+  </div>
+</template>
+
+<style scoped>
+h1 {
+  text-align: center;
+}
+.data-box {
```

Comment:
Omdat je gebruik maakt van de scoped styles van vue, zou je in deze stijlregel rechtstreeks de `table` kunnen targetten en heb je die div niet nodig